src/mongo/jobs/ApplyOperation.class.php

```diff
@@ -64,6 +86,6 @@ protected function createImpactedSubject(array $args)
      */
     protected function getMandatoryArgs()
     {
-        return array("tripodConfig","subject");
+        return array("tripodConfig","subjects");
```

Comment:
v minor , but should these be constants?  there's a few places where we use these keys